src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import { ComponentStory } from "@storybook/react"
+import { FC } from "react"
+import { placements } from "./placements"
+import { Tooltip } from "./Tooltip"
+
+/**
+ * See Storybook Docs: Writing Stories
+ * https://storybook.js.org/docs/basics/writing-stories/
+ */
+
+const tooltipDecorator = (Story: FC) => (
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "center",
+      margin: "5rem",
+    }}
+  >
+    <Story />
+  </div>
+)
+
+export default {
+  title: "Components/Tooltip",
+  decorators: [tooltipDecorator],
+  argTypes: {
+    placement: {
+      control: {
+        type: "select",
+        options: placements,
+      },
+    },
+  },
+}
+
+const Template: ComponentStory<typeof Tooltip> = (args) => {
+  return <Tooltip {...args} />
+}
+
+export const Default = Template.bind({})
+
+Default.args = {
+  children: "Hello",
+  label: "World",
+  distance: 10,
+  skidding: 0,
+  placement: "bottom",
+  showArrow: false,
+  enterDelay: 0,
+  exitDelay: 0,
+  hideOnClick: false,
+  alwaysShow: false,
+}
+
+export const WithArrow = Template.bind({})
+
+WithArrow.args = {
+  ...Default.args,
+  alwaysShow: true,
+  showArrow: true,
+}
+
+export const WithoutArrow = Template.bind({})
+
+WithoutArrow.args = {
+  ...Default.args,
+  alwaysShow: true,
+}
+
+export const WithAdditionalContent: ComponentStory<typeof Tooltip> = (args) => {
+  return (
+    <div
+      style={{
+        lineHeight: "24px",
```

Comment:
⛏️ I was surprised to see the line-height used for the paragraph affecting the tooltip's line-height. Would it make sense for `.box` to have its own default line-height?

<img width="486" alt="Screen Shot 2022-08-15 at 4 14 41 PM" src="https://user-images.githubusercontent.com/157195/184720086-3759b247-ff26-4b79-a37a-0fd7510c0431.png">